prqlc/prqlc-parser/Cargo.toml

```diff
@@ -10,6 +10,7 @@ version.workspace = true
 
 [lib]
 doctest = false
+test = false
```

Comment:
We lost some tests here — https://github.com/PRQL/prql/blob/a9d81a429b6a2c940e94fcf177c402842c05f25b/prqlc/prqlc-parser/src/lexer.rs#L507

Should we move them or enable them here?

When working on a sub-project, it can be nice to have the tests loop be extremely fast, because we only need to compile that crate (and plausibly have the tests next to the code?). OTOH it adds to the number of binaries we need to create to run all the tests (I won't link to the blog post this time :) )

Reply:
These tests are still run with `cargo test --lib` or `cargo test --all-targets`.

I've left them here, because they cannot be tested via public API of prqlc-parser crate (well, it could be done probably, but they would have to be rewritten a bit).

Reply:
> I've left them here, because they cannot be tested via public API of prqlc-parser crate (well, it could be done probably, but they would have to be rewritten a bit).

What's the principle here? Are we excluding internal-only tests by default?